metadata-ingestion/setup.py

```diff
@@ -311,6 +311,9 @@
     # https://www.elastic.co/guide/en/elasticsearch/client/python-api/current/release-notes.html#rn-7-14-0
     # https://github.com/elastic/elasticsearch-py/issues/1639#issuecomment-883587433
     "elasticsearch": {"elasticsearch==7.13.4"},
+    "cassandra": {
+        "cassandra-driver~=3.28.0"
```

Comment:
I read somewhere we should avoid pinning versions if possible. Should i remove this?
historically I've pinned versions so that we only intentionally upgrade package versions. This way nothing breaks unexpectedly.